card.cpp

```diff
@@ -0,0 +1,8 @@
+#include "card.hpp"
+#include <iostream>
+
+Card::Card(std::string s, std::string v) : suit(s), value(v) {}
+
+void Card::displayCard() {
```

Comment:
Maybe we should move the in/out operations to outside the model? Better to create the getters and setters and leave this responsibility to the screen.

Reply:
Done